e2e/integration/utils/config.go

```diff
@@ -0,0 +1,88 @@
+/*
+Copyright 2019 The KubeEdge Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package utils
+
+import (
+	"encoding/json"
+	"math/rand"
+	"os"
+	"path/filepath"
+	"time"
+)
+
+//config.json decode struct
+type Config struct {
+	EdgedEndpoint string   `json:"edgedEndpoint"`
+	AppImageUrl   []string `json:"image_url"`
+	ApiServer     string   `json:"apiserver"`
+}
+
+//config struct
+var config *Config
+
+//get config.json path
+func LoadConfig() Config {
+	if config == nil {
+		config = loadConfigJsonFromPath()
+	}
+	return *config
+}
+
+//Load Config.json from the PWD, and decode the config.
+func loadConfigJsonFromPath() *Config {
+	path := getConfigPath()
+	_, err := filepath.Abs(filepath.Dir(path))
+	if err != nil {
+		InfoV6("Failed to get Abs path: %v", err)
+		panic(err)
+	}
+	var config *Config = &Config{}
+	configFile, err := os.Open(path)
+	if err != nil {
+		panic(err)
+	}
+	decoder := json.NewDecoder(configFile)
+	err = decoder.Decode(config)
+	if err != nil {
+		panic(err)
+	}
+	return config
+}
+
+//Get config path from Env or hard code the file path
+func getConfigPath() string {
+	path := os.Getenv("TESTCONFIG")
+	if path == "" {
+		path = "config.json"
+	}
+	return path
+}
+
+func RandomInt(min, max int) int {
```

Comment:
Pls add a comment with a short description above this function explaining its functionality